impl/client/src/main/javascript/web/pentaho/visual/impl/View.js

```diff
@@ -388,7 +388,8 @@ define([
       }
 
       return this;
-    }
+    },
+
```

Comment:
Dangling comma. Also, does this file have any relevant changes?

Reply:
There were changes from a previous commit, the comma was left by mistake. The file should no longer have any changes